docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+version: '3'
+
+services:
+  database:
+    container_name: signsfive_database
+    image: mariadb
+    volumes:
+      - ./db_data:/var/lib/mysql
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: signsfive_dev
+      MYSQL_USER: signsfive_dev
+      MYSQL_PASSWORD: signsfive_dev
+    networks:
+      signsfive_net:
+        ipv4_address: 172.28.1.1
+
+  api:
+    container_name: signsfive_api
+    depends_on:
+      - database
+    image: node:alpine
+    user: node
+    working_dir: /home/node/app
+    environment:
+      - NODE_ENV=development
+      - PORT_API=8282
+    volumes:
+      - ./db:/home/node/app/db
+      - ./node_modules:/home/node/app/node_modules
+      - ./scripts:/home/node/app/scripts
+      - ./src:/home/node/app/src
+      - ./.npm:/home/node/.npm
+      - ./.sequelizerc:/home/node/app/.sequelizerc
+      - ./package.json:/home/node/app/package.json
+      - ./package-lock.json:/home/node/app/package-lock.json
+      - ./server.log:/home/node/app/server.log
+    ports:
+      - 8999:8282
+    command: npm start
+    networks:
+        signsfive_net:
+          ipv4_address: 172.28.1.2
```

Comment:
how did you decide on ip address?

Reply:
This is now in `.env` and only applies to development mode.